src/Fixer/Operator/StringConcatenationToInterpolationFixer.php

```diff
@@ -0,0 +1,614 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Operator;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Jean LECORDIER <jean.lecordier@infoclimat.fr>
+ */
+final class StringConcatenationToInterpolationFixer extends AbstractFixer
+{
+    public function getName(): string
+    {
+        return 'Infoclimat/string_concatenation_to_interpolation';
+    }
+
```

Comment:
```suggestion
```

This have to be removed, built-in fixers have different naming convention.

Reply:
Yup, just seen that myself, sorry for that